junit-platform-commons/src/main/java/org/junit/platform/commons/util/StringUtils.java

```diff
@@ -180,4 +183,15 @@ public static String nullSafeToString(Object obj) {
 		}
 	}
 
+	/**
+	 * Replace all ISO control characters in the supplied {@link String}.
+	 *
+	 * @param str the string to check; may be {@code null}
+	 * @param replacement the replacement string; never {@code null}
+	 * @return The supplied string with all control characters replaced.
+	 */
+	public static String replaceIsoControlCharacters(String str, String replacement) {
```

Comment:
:x: Missing the `@since` tag in the Javadoc and the `@API(status = INTERNAL, since = "1.4")` annotation.

Reply:
✔️